stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -942,15 +942,15 @@ internal class ChannelControllerImpl(
             val result = domainImpl.runAndRetry { client.deleteReaction(reaction.messageId, reaction.type) }
             return if (result.isSuccess) {
                 reaction.syncStatus = SyncStatus.COMPLETED
-                domainImpl.repos.insertReaction(reaction)
```

Comment:
Why do you remove the insertion into our database?

Reply:
This code gets called in the line 931 and there's no condition for it to run, so this line is just a duplication.

Reply:
But it update the `syncStatus` to be updated on the DB

Reply:
Oh, I didn't realise that this new insertReaction is to propagate the new sync status.